dockstore-client/src/main/java/io/dockstore/common/FileProvisioning.java

```diff
@@ -285,7 +284,7 @@ private void provisionInputFile(String parameterFilePath, String targetPath, Pat
                         }
                         Files.createLink(localPath, potentialCachedFile);
                     } catch (IOException e) {
-                        LOG.error("Cannot create hard link to cached file, you may want to move your cache", e.getMessage());
+                        LOG.error("Cannot create hard link to cached file, you may want to move your cache \n" + e.getMessage());
```

Comment:
I would log the exception, either as well or instead of the string change, e.g., `LOG.error("message", e)` -- stack trace is useful when looking at logs

Reply:
Missed this but targeted on next cleanup run. Don't be afraid to request changes even if its small :) :medal_military: